README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## This is my brief Biography!
+
+My name is Eleanor Anderson. I am a current fourth year undergraduate student majoring in Economics and Psychology. I am planning on graduating this quarter and am applying to graduate schools. I wanted to take this class becuase I have had a handful of classes that use R but I never truly learned how R works, I mostly learned how to plug and chug. I am **very** *excited* to learn how to actually use R!
+
+My main hobbies and interests are as follows:
+
+* Dance
+* Music (both playing and listening)
+* Cooking and Baking
+* Knitting 
+
+### Dance
+
+Dance is *probably* what I spend the majority of my time doing! It is so freeing and I get to spend time with some of the **greatest** people I know! I have attached a picture of my dance group below:
```

Comment:
I can’t see your dance group picture. I didn’t add any picture to my assignment, but I great tool I use was the R markdown cheat sheet! Here is the link: https://rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf